### DISCUSSION TITLE
Rotating text

### DISCUSSION THREAD

**Discussion by eb-64-64** [bevyengine/bevy#3161] [Q&A]
Looking at this code:
```rust
commands.spawn_bundle(Text2dBundle {
    text: Text::with_section(
        "Blah",
        TextStyle {
            font,
            font_size: FONT_SIZE,
            color: FONT_COLOR,
        },
        TextAlignment {
            vertical: VerticalAlign::Center,
            horizontal: HorizontalAlign::Center,
        },
    ),
    transform: Transform {
        rotation: Quat::from_rotation_z(std::f32::consts::PI),
        ..Default::default()
    },
    ..Default::default()
});
```
One would expect the text to be rendered upside-down (that is, flipped 180° or π radians). However, that's not the case. Looking at the source for `Text2dBundle`, there's a comment that says that the `rotation` of the `Transform` is ignored:
> Position is determined by the `Transform`'s translation, though scale and rotation are ignored.

Is it possible to render text that is rotated, either with `Text2dBundle` or some other text entity/bundle?

_Edit: this was originally a Discord discussion, see that [here](https://discordapp.com/channels/691052431525675048/691052431974465548/911868658987462656)._

**Comment by djeedai** [comment]
If I understand correctly your question, I believe there's a bug that was fixed in #2103. This is not yet available in any released version though, will be in the 0.6 I believe.
